services/common/src/components/permits/PermitReviewBanner.tsx

```diff
@@ -1,55 +1,76 @@
+}
 
 export const PermitReviewBanner: FC<PermitReviewBannerProps> = ({
-    isExtracted,
-    height,
-    isReviewComplete
+  isExtracted,
+  height,
+  isReviewComplete,
 }) => {
-    const completeParams = {
-        text: "Conditions were extracted using AI and have been reviewed and verified.",
-        className: "complete",
-        icon: faInfoCircle
+  const isCore = useAppSelector(getIsCore);
+
+  const completeParams = {
+    text: "Conditions were extracted using AI and have been reviewed and verified.",
+    className: "complete",
+    icon: faInfoCircle,
+  };
+
+  const mineSpaceParams = {
+    text: "Permit conditions have been extracted using AI and reviewed by inspector. These conditions are for reference only and cannot be used as legal requirements. The issued permit remains the official source of truth.",
+    className: "incomplete",
+    icon: faWarning,
+  };
+
+  const incompleteParams = {
+    text: "Conditions and their report requirements have been extracted using AI and require review and verification.",
+    className: "incomplete",
+    icon: faWarning,
+  };
+
+  const draftedCoreParams = {
+    ...completeParams,
+    text: "This permit was drafted and issued in Core; conditions cannot be modified.",
+  };
+
+  const paramsToUse = useMemo(() => {
+    if (!isCore) {
+      return mineSpaceParams;
     }
+    if (isExtracted) {
+      return isReviewComplete ? completeParams : incompleteParams;
+    } else {
+      return draftedCoreParams;
+    }
+  }, [isExtracted, isReviewComplete, isCore]);
 
-    const incompleteParams = {
-        text: "Conditions and their report requirements have been extracted using AI and require review and verification.",
-        className: "incomplete",
-        icon: faWarning
-    };
-
-    const draftedCoreParams = {
-        ...completeParams,
-        text: "This permit was drafted and issued in Core; conditions cannot be modified."
-    };
-
-    const aiParams = isReviewComplete ? completeParams : incompleteParams;
-    const paramsToUse = isExtracted ? aiParams : draftedCoreParams;
-
-    return (<>
-        <Row
-            justify="center"
-            align="middle"
-            className={`permit-status-banner permit-status-banner--${paramsToUse.className}`}
-            style={{
-                height,
-            }}
-        >
-            <Col>{<FontAwesomeIcon icon={paramsToUse.icon} />}</Col>
-            <Col>{paramsToUse.text}</Col>
-        </Row>
-        <div
-            style={{
-                height
-            }}
-        ></div>
-    </>);
-};
\ No newline at end of file
+  return (
+    <>
+      <Row
+        justify="center"
+        align="middle"
+        gutter={16}
+        className={`permit-status-banner permit-status-banner--${paramsToUse.className}`}
+        style={{
+          height,
+        }}
+      >
+        <Col>{<FontAwesomeIcon icon={paramsToUse.icon} />}</Col>
+        <Col span={23}>{paramsToUse.text}</Col>
```

Comment:
Just checking- visually looks the same/better in Core as before?

Reply:
Looks the same for the most part.  Could end up off by a few pixels, but without it the long text was taking up the whole width and leaving the icon on it's own row.